src/Healthchecks.php

```diff
@@ -360,4 +358,39 @@ private function getCheck(string $key)
 	{
 		return $this->getChecks()[$key] ?? null;
 	}
+
+	/**
+	 * @param array[][] $queryParams
+	 *
+	 * @return string
+	 */
+	private function buildQuery(array $queryParams = []): string
+	{
+		array_walk(
```

Comment:
I don't remember whether we could use [http_build_query](http://php.net/manual/pl/function.http-build-query.php) or not. You had issues in composing arrays which was a blocker, right?
`&x=1&x=2&x=3` vs `&x[]=1&x[]=2&x[]=3`

Reply:
Yeap unfortunately, that was and is the problem :(